data/174/627/702/5/1746277025.geojson

```diff
@@ -0,0 +1,55 @@
+{
+  "id": 1746277025,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.888408,
+    "geom:bbox":"-115.217686,31.030894,-113.610451,32.158876",
+    "geom:latitude":31.521451,
+    "geom:longitude":-114.501951,
+    "iso:country":"MX",
+    "lbl:latitude":31.452968,
+    "lbl:longitude":-114.648142,
+    "mps:latitude":31.452968,
+    "mps:longitude":-114.648142,
+    "mx-conanp:id_anp":"2.1.01.104",
+    "mx-conanp:name":"Alto Golfo de California y Delta del R\u00edo Colorado",
+    "mx-conanp:type":"Biosphere Reserve",
+    "mx-conanp:type_es":"Reserva de la Biosfera",
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_landuse_aoi":1,
+    "name:spa_x_preferred":[
+        "Alto Golfo de California y Delta del R\u00edo Colorado"
+    ],
+    "src:geom":"mx-conanp",
+    "src:lbl_centroid":"mapshaper",
+    "wof:belongsto":[],
+    "wof:breaches":[],
+    "wof:concordances":{
+        "mx-conanp":"2.1.01.104"
+    },
+    "wof:country":"MX",
+    "wof:created":1630532118,
+    "wof:geomhash":"9eca59106c7edbfc834b957efc9f1ac4",
+    "wof:hierarchy":[],
+    "wof:id":1746277025,
+    "wof:lastmodified":1630532118,
+    "wof:name":"Alto Golfo de California y Delta del R\u00edo Colorado",
```

Comment:
Generally we prefer ASCII in the `wof:name`, maybe revisit this once you have English versions of the name to use?

Reply:
Yeah, agreed.. I'll spend some time harvesting Wikidata ids for these new features, then updating these names with the English translation.

Reply:
Whenever possible, I've updated these wof:name values using Wikidata imports.